eq-author-api/schema/tests/page.test.js

```diff
@@ -50,6 +50,29 @@ describe("page", () => {
         position: 0,
       });
     });
+
+    it("should create at a given position", async () => {
+      questionnaire = await buildQuestionnaire({
+        sections: [
+          {
+            pages: [{}],
+          },
+        ],
+      });
+      const section = questionnaire.sections[0];
+
+      const createdPage = await createQuestionPage(questionnaire, {
+        title: "Title",
+        description: "Description",
+        sectionId: section.id,
+        position: 0,
+      });
+      expect(createdPage).toMatchObject({
+        title: "Title",
+        description: "Description",
+        position: 0,
+      });
```

Comment:
Instead of asserting on the object that's created, just wondering if the test could be a little more explicit and query the section and then assert that the first page is correct?